snap/snapcraft.yaml

```
@@ -40,15 +40,14 @@ parts:
 
 apps:
   brave:
-    command: bin/desktop-launch $SNAP/usr/bin/brave --no-sandbox
+    command: bin/desktop-launch $SNAP/opt/brave.com/brave/brave  --no-sandbox
```

your previous PR removed the desktop file. Please add it back with the code that massages the Icon path.

---

Added back!

---

thanks for that. We also need to tell snapcraft where the desktop file is:

```suggestion
    command: bin/desktop-launch $SNAP/opt/brave.com/brave/brave  --no-sandbox
    desktop: usr/share/applications/brave-browser.desktop
```